addons/website_event_track/static/src/js/website_event_track_proposal.js

```diff
@@ -0,0 +1,36 @@
+odoo.define('website_event_track.website_event_track_proposal', function (require) {
+'use strict';
+
+var publicWidget = require('web.public.widget');
+var weDefaultOptions = require('web_editor.wysiwyg.default_options');
+var wysiwygLoader = require('web_editor.loader');
+
+publicWidget.registry.websiteEventTrackProposal = publicWidget.Widget.extend({
+    selector: '.o_website_event_talk_proposal',
+
+    start: function() {
+        var self = this;
+        _.each($('textarea.o_wysiwyg_loader'), function (textarea) {
```

Comment:
2 things here.

1.
You should place yourself after the "start" to make sure everything is properly loaded.
`return this._super.apply(this, arguments).then(function() { ... });`
Or better yet extract to another function to make it more readable:
`return this._super.apply(this, arguments).then(this.loadWysiwyg.bind(this));`

2.
Avoid global selectors whenever possible.
Use `this.$('textarea.o_wysiwyg_loader');`